### DISCUSSION TITLE
Card Payment Brick, disabled external button

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-react#45] [Q&A] [answered]
Hi, im using an external button to send payment information from CardPayment, everything works just fine, but i have question, is there a way to know if the form is completed?, Cause i want to toggle my button from a disabled state only if the form is completed
Thanks

**Comment by lemazzo** [comment, marked as answer]
Hi @joxpulp!

Bricks doesn't have this feature right now, but it's interesting and we'll add it to backlog.
We'll let you know when it's available!

Thanks for sharing
